cli/command/container/opts.go

```diff
@@ -190,6 +190,7 @@ func addFlags(flags *pflag.FlagSet) *containerOptions {
 	flags.Var(&copts.securityOpt, "security-opt", "Security Options")
 	flags.StringVar(&copts.usernsMode, "userns", "", "User namespace to use")
 	flags.StringVar(&copts.credentialSpec, "credentialspec", "", "Credential spec for managed service account (Windows only)")
+	flags.SetAnnotation("credentialspec", "os", []string{"windows"})
```

Comment:
nit: Should we name this "ostype" / "os-type" as well to be consistent (otherwise there's a risk they're getting different meanings)?